### PR TITLE
Tracing registration

### PR DESCRIPTION
**Pull request by DanRoseus**
Auto enable tracing for MQ/APIC if tracing installed
Auto approve the tracing registration and create the required secrets

### CODE REVIEW

File: products/bash/release-products.sh [hunk 1]
```diff
@@ -93,6 +93,94 @@ chart_repo=ibm-entitled-charts
...
+        exit 1
+      fi
+  		echo "Waiting for tracing registration job to complete (Attempt $i of 60). Job status: $TRACING_JOB_STATUS"
+  		kubectl -n ${REG_NAMESPACE} get pods,jobs
+  		echo "Checking again in one minute..."
+  		sleep 60
+  	fi
+  done
+
+  echo "Waiting 1 minute for tracing registration to trickle through"
```
Comment (hrizvi1) [inline products/bash/release-products.sh:124]: Do we want to be funny here? :)

File: products/bash/release-products.sh [hunk 2]
```diff
@@ -93,6 +93,94 @@ chart_repo=ibm-entitled-charts
...
+  	if [[ -z "$TRACING_POD" ]]; then
+      echo "Waiting for tracing pod (Attempt $i of 60), checking again in 15 seconds..."
+  		sleep 15
+  	else
+      echo "Got tracing pod: ${TRACING_POD}"
+  		break
+  	fi
+  done
+  if [[ -z "$TRACING_POD" ]]; then
+    echo "Failed to get tracing pod"
```
Comment (hrizvi1) [inline products/bash/release-products.sh:140]: should we fail here or should we ask the user to do the setup manually instead of exiting?

Comment (DanRoseus) [reply products/bash/release-products.sh:140]: If this tracing registration fails for APIC then APIC gateway will fail to start and the APIC configuration step won't work. I think it's hosed if this doesn't work, so just exit

File: products/bash/release-products.sh (hunk 1 above)
Comment (DanRoseus) [reply products/bash/release-products.sh:124]: Not sure what you're asking for!

Comment (hrizvi1) [reply products/bash/release-products.sh:124]: something like: Waiting for tracing registration to complete

File: products/bash/release-products.sh (hunk 2 above)
Comment (hrizvi1) [reply products/bash/release-products.sh:140]: ok

File: products/bash/release-products.sh (hunk 1 above)
Comment (DanRoseus) [reply products/bash/release-products.sh:124]: Ah, ok, I'm happy with "trickle through"

**Pull request merged**